basic-examples/expressions/config.yaml

```diff
@@ -0,0 +1,8 @@
+deployers:
+  image:
+    deployer_name: docker

```

Comment:
The other examples have all been converted recently to defaulting to podman as the deployer. We should be consistent on this.